src/RecoveryServices/RecoveryServices.Backup.Autorest/custom/Helpers/PolicyHelpers.ps1

```diff
@@ -220,11 +220,11 @@ function ValidateRetentionParameters {
             $parametersEntered = [System.Collections.ArrayList]::new()
 			if ($MonthlyRetentionScheduleType) 
             {
-				$parametersEntered.Add("MonthlyRetentionScheduleType")
+				$parametersEntered.Add("MonthlyRetentionScheduleType") > $null
```

Comment:
why do we need > $null ?

Reply:
it is to supress the unwanted array that was coming 
$parametersEntered.Add("MonthlyRetentionScheduleType") alone was returning the index of the added param